htdocs/product/class/product.class.php

```diff
@@ -825,10 +825,10 @@ public function create($user, $notrigger = 0)
 					$sql .= ", label";
 					$sql .= ", fk_user_author";
 					$sql .= ", fk_product_type";
+					$sql .= ", price_label";
```

Comment:
Old code seems good.
In values we have
```
$sql .= ", ".((int) $user->id);
					$sql .= ", ".((int) $this->type);
					$sql .= ", ".price2num($price_ht, 'MT');
					$sql .= ", ".price2num($price_ttc, 'MT');
```
so after fk_product_type, we should have price.

Reply:
Now it's  working, looks like it's been solved right before I posted the pull request.